### PR TITLE
fix log sidecar container docs

### PR DESCRIPTION
**Pull request by zmberg**

### CODE REVIEW

File: docs/best-practices/log-container-sidecarset.md [hunk 1]
```diff
@@ -133,17 +130,19 @@ spec:
...
-    # Share log directory with app container via volumeMount
-    - mountPath: /var/log
-      name: log
-    volumes:
-    - configMap:
-        name: filebeat-config
-      name: filebeat-config
-    - name: log
-      emptyDir: {}
+    name: config
```
Comment (furykerry) [inline docs/best-practices/log-container-sidecarset.md:143]: still indentation problem here

File: docs/best-practices/log-container-sidecarset.md [hunk 2]
```diff
@@ -91,22 +91,19 @@ metadata:
   name: filebeat-sidecarset
 spec:
   selector:
-    # Pod labels that need to be injected into the sidecar container
```
Comment (furykerry) [inline docs/best-practices/log-container-sidecarset.md:94]: there is also indentation problem in L33, around the line 
 ### K8S Sidecar Model Disadvantage

in addition, please add the resource declaration in the code blocks(add apiserver, kind part)

File: docs/best-practices/log-container-sidecarset.md (hunk 1 above)
Comment (zmberg) [reply docs/best-practices/log-container-sidecarset.md:143]: There's no problem here.

**Pull request merged**